src/Illuminate/Testing/TestResponse.php

```diff
@@ -1361,7 +1362,11 @@ public function dump()
             $content = $json;
         }
 
-        dump($content);
+        if ($key) {
```

Comment:
shoiuld be a null check here, otherwise people can't dump key `0`

Reply:
Good catch! Fixed.